From 53248e7627e910d7d10e28363c90dbb791d5358b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:14:04 -0400
Subject: [cpufreq] powernv: make local function static

Message-id: <1410545655-205645-616-git-send-email-dzickus@redhat.com>
Patchwork-id: 94461
O-Subject: [RHEL7 PATCH 615/626] cpufreq: powernv: make local function static
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 60d1ea4e0aaeedb89c4336eb4d1359bee2cb33aa
Author: Brian Norris <computersforpeace@gmail.com>
Date:   Sun May 11 00:51:20 2014 -0700

    cpufreq: powernv: make local function static

    powernv_cpufreq_get() is only referenced in this file.

    Signed-off-by: Brian Norris <computersforpeace@gmail.com>
    Reviewed-by: Srivatsa S. Bhat <srivatsa.bhat@linux.vnet.ibm.com>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org> on V2.
    Reviewed-by: Gautham R. Shenoy <ego@linux.vnet.ibm.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/powernv-cpufreq.c b/drivers/cpufreq/powernv-cpufreq.c
index af49688..bb1d08d 100644
--- a/drivers/cpufreq/powernv-cpufreq.c
+++ b/drivers/cpufreq/powernv-cpufreq.c
@@ -235,7 +235,7 @@ static void powernv_read_cpu_freq(void *arg)
  * firmware for CPU 'cpu'. This value is reported through the sysfs
  * file cpuinfo_cur_freq.
  */
-unsigned int powernv_cpufreq_get(unsigned int cpu)
+static unsigned int powernv_cpufreq_get(unsigned int cpu)
 {
  struct powernv_smp_call_data freq_data;
 
-- 
1.7.1